app/assets/stylesheets/styles/base/_icons.scss

```diff
@@ -118,6 +126,22 @@
     }
   }
 
+  &--redlist {
+    @include flex;
+    @include flex-center;
+    background-color: $iucn;
+    border-radius: 100px;
```

Comment:
100% - 100px is fine here, but I guess wouldn't scale if you were to make these big